### ISSUE TITLE
Add ability to edit and/or delete keys

### ISSUE THREAD

**Issue by phavlik** [mpuittinen/yaml-edit#1]
Would be nice to have ability to edit the keys, or at least remove them (as then I can do edit as delete + insert).

**Comment by mackbrowne** [comment]
@phavlik yeah im a little blown away.  Why didn't you name this plugin yaml-insert ?
